src/components/NotificationBox.tsx

```diff
@@ -17,19 +17,21 @@ const NotificationBox = ({ style, notificationText }: Props): JSX.Element => (
           justifyContent: 'center',
           alignItems: 'center',
           flexWrap: 'wrap',
-          padding: suomifiTheme.spacing.l,
-          margin: `${suomifiTheme.spacing.m} 0`,
+          padding: suomifiDesignTokens.spacing.l,
+          margin: `${suomifiDesignTokens.spacing.m} 0`,
           ...style,
-          background: suomifiTheme.colors.accentSecondaryLight40,
+          background: suomifiDesignTokens.colors.accentSecondaryLight40,
           fontSize: '24px',
         }}
       >
-        <Icon icon="error" color={suomifiTheme.colors.accentSecondary} />
+        <Icon icon="error" color={suomifiDesignTokens.colors.accentSecondary} />
         <Text.bold
           style={{
-            marginLeft: suomifiTheme.spacing.m,
+            marginLeft: suomifiDesignTokens.spacing.m,
             verticalAlign: 'middle',
-            fontSize: suomifiTheme.typography.fontSize.body,
+            fontSize:
+              suomifiDesignTokens.values.typography.bodyText.fontSize.value +
+              suomifiDesignTokens.values.typography.bodyText.fontSize.unit,
```

Comment:
I think you can just use `fontSize: suomifiDesignTokens.typography.bodyText` here. The readme of the token repo might not be quite up to date about this.

Reply:
Checked it, and it seems that your way seems to be the best one at this point after all.